sailor/pai/utils.py

```diff
@@ -0,0 +1,21 @@
+"""Module for various utility functions, in particular those related to fetching data from remote oauth endpoints."""
+
+import logging
+
+from ..utils.config import SailorConfig
+from ..assetcentral.utils import AssetcentralEntity
+LOG = logging.getLogger(__name__)
+LOG.addHandler(logging.NullHandler())
```

Comment:
remove everything related to logging, as it is not used.

Reply:
done